sed/calibrator/energy.py

```diff
@@ -2312,12 +2313,10 @@ def poly_energy_calibration(
     ecalibdict["Tmat"] = t_mat
     ecalibdict["bvec"] = bvec
     ecalibdict["energy_scale"] = energy_scale
+    ecalibdict["E0"] = -(pfunc(-1 * ref_energy, pos[0]) + vals[0])
```

Comment:
Similar considerations here.

Reply:
don't really see a need for a partial function here when it's only one argument. But I understand if it's there for a reason

Reply:
I'll look into this eventually. It's some old code.